index.js

```diff
@@ -48,6 +56,11 @@ console.log('hello world')
 
 // TODO: ADD CODE HERE
 
+let price = "8.99"
+let quantity = "2"
+let total = price * quantity;
+console.log( total);
```

Comment:
good start here. You'll also want to round the total to 2 decimal points.
Try looking up the javasript method `toFixed ()`